src/lib/holocene/api-pagination.svelte

```diff
@@ -226,7 +226,7 @@
       <Input
         icon="search"
         id="api-pagination-search-input"
-        class="grow"
+        class="w-full grow sm:w-1/2"
```

Comment:
Other width classes aren't needed here with the addition of `grow`

```suggestion
        class="grow"
```